core-services/image-mirroring/lvms/mapping_lvms_latest

```diff
@@ -0,0 +1,2 @@
+registry.ci.openshift.org/ocp/lvm-operator:* quay.io/lvms_dev/lvms-operator
```

Comment:
Can we also mirror bundle and catalog images?

Reply:
AFAIK we dont promote the bundle images because they contain the images from the bundle. We would need to introduce a new image for the bundle first